### ISSUE TITLE
[Send 2][Enhancement] Clicking inside the “Privacy” overlay should dismiss it

### ISSUE THREAD

**Issue by Softvision-AndreeaCupsa** [mozilla/send#1304]
**[Notes]:**
- There should be consistency regarding the dismissal of the overlay considering that the upload complete and Sign in/up overlays are dismissed when clicking inside them.

**[Affected versions]:**
- Firefox 66.0.3 and up
- Chrome Version 73.0

**[Affected Platforms]:**
- All Mac
- All Windows
- All Linux

**[Steps to reproduce]:**
1. Open the browser and navigate to the https://send.firefox.com/ website.
2. Click the “Privacy” button.
3. Click inside the overlay but not on any actionable buttons.

**[Expected result]:**
- The overlay is dismissed.

**[Actual result]:**
-  The overlay is still displayed.

**[Additional Notes]:**
- This issue is also reproducible on the https://send2.dev.lcip.org website.
- Attached a screen recording with the issue:

![rec of issue - overlay dismissal](https://user-images.githubusercontent.com/37875464/56498368-09341f80-650a-11e9-96d2-6f26b397b128.gif)

**Comment by TheBolliwood** [comment]
The link to "Privacy" isn't an overlay like the Signup Modal, its another page (https://send.firefox.com/legal).
In my opinion you shouldn't be able to go back to the main page just by clicking inside the white space there.
This page doesn't breaks consistency because you also can't dismiss other pages like `/unsupported`, `/error` or  `/download`.

**Comment by allen505** [comment]
The back button here seems to be the Firefox send icon(as seen in the Screen record). So it does maintain the consistency.

**Comment by blenderskool** [comment]
Can I try working on this? 😄 @johngruen

**Comment by M4trox** [comment]
I don't think that clicking inside "Privacy" overlay should result in overlay dismiss. Someone may want to select some text, copy and paste it elsewhere - for example, when explaining Firefox Send to a friend. What's interesting is that now the text selection is disabled by CSS (`user-select: none`). It should be changed, too, but not necessarily in this issue.

I propose that we should add a link at the bottom that says e.g. `Go back to Firefox Send homepage`. Considering the way the site is designed, I think that the most appropriate way to do this is to use blue link.

![screenshot-2019-06-06 20_41_52](https://user-images.githubusercontent.com/3310268/59057844-a0c3b680-889b-11e9-8045-5f704bc51efa.png)

**The goal should be to help user easily navigate to Firefox Send homepage.** Adding link will also help with accessibility - screen reader or voice over can read this link.

**Comment by Prateek93a** [comment]
Hello world, can I, @Prateek93a, work on this one as my first contribution to Mozilla community?
How I look at this, is that if we have a closer look at all other links beside Privacy, such as Legal, Cookies and Github, they all point to different links and take the user to entirely different pages. On the  other hand, Privacy link sort of opens a modal like page which very similar to sendFile page, without a back button which can confuse users into thinking that its just an overlay and there should be a way to go back.
Hence what I propose is that we can have Privacy button placed somewhere else so as to separate it from other links and implement 'click to dismiss overlay' feature or as @M4trox suggested, we can have a back button placed inside it so that user doesn't get confused.

**Comment by sharmaaditya570191** [comment]
Hi @dannycoates . I would like to take this up.Could you please guide me as to how to implement this?
